static/app/views/organizationGroupDetails/header.tsx

```diff
@@ -195,6 +325,8 @@ class GroupHeader extends Component<Props, State> {
       </GuideAnchor>
     );
 
+    const isPerformanceIssue = !!event?.contexts?.performance_issue;
```

Comment:
can you add a TODO here to update this since going forward groups would return a `type` and that would determine what kind of issue a group is